html_text/tests/test_extractor.py

```diff
@@ -0,0 +1,53 @@
+# Copyright 2016-2017 Jairo Llopis <jairo.llopis@tecnativa.com>
```

Comment:
everywhere update to 2018

Reply:
Not for this contributor. He hasn't done anything in this year.